.gitlab-ci.yml

```diff
@@ -19,6 +19,58 @@ include:
   - project: 'Northern.tech/Mender/mendertesting'
     file: '.gitlab-ci-check-docker-deploy.yml'
 
+test:acceptance_tests:
+  stage: test
+  tags:
+    - docker
+  image: tiangolo/docker-with-compose
+  services:
+    - docker:19.03.5-dind
+  script:
+    - make acceptance-testing-build
+    - make acceptance-testing-up
+    - sh -c "sleep 30"
+    - make acceptance-testing-run
+  after_script:
+    - make acceptance-testing-logs
+    - make acceptance-testing-down
+  artifacts:
+    expire_in: 2w
+    paths:
+      - tests/coverage-acceptance.txt
+    when: always
+
+publish:acceptance:
+  stage: publish
+  except:
+    - /^saas-[a-zA-Z0-9.]+$/
+  image: golang:1.14-alpine3.11
+  needs:
+    - job: test:acceptance_tests
+      artifacts: true
+  before_script:
+    - apk add --no-cache git
+    # Run go get out of the repo to not modify go.mod
+    - cd / && go get github.com/mattn/goveralls && cd -
+    # Coveralls env variables:
+    #  According to https://docs.coveralls.io/supported-ci-services
+    #  we should set CI_NAME, CI_BUILD_NUMBER, etc. But according
+    #  to goveralls source code (https://github.com/mattn/goveralls)
+    #  many of these are not supported. Set CI_BRANCH, CI_PR_NUMBER,
+    #  and pass few others as command line arguments.
+    #  See also https://docs.coveralls.io/api-reference
+    - export CI_BRANCH=${CI_COMMIT_BRANCH}
+    - export CI_PR_NUMBER=${CI_COMMIT_BRANCH#pr_}
```

Comment:
This is being modified as we speak. See https://github.com/mendersoftware/mendertesting/pull/152#discussion_r653600804